src/amo/components/Addon/index.js

```diff
@@ -60,6 +60,14 @@ import ShowMoreCard from 'ui/components/ShowMoreCard';
 import './styles.scss';
 
 
+// Find out if slug is invalid.
+const isInvalidSlugID = (slug) => {
```

Comment:
The slug is not really invalid actually, it is "just" not a string no? If so, we should probably update the function name.

Reply:
hmm yeh i'll take another look at the name, thanks

Reply:
actually from my understanding it could be a number type (legacy) or it could be a string which looks like a number (1234) - either of these cases should redirect. The problem was there are some strings like '-1234' which look like a number but we need to exclude these cases (not redirect)... so i am not sure of a better name - maybe `isSlugValidNumber` ?

Reply:
How about `slugIsID`?